src/components/UI/backdrop/index.js

```diff
@@ -5,7 +5,9 @@ import PropTypes from 'prop-types';
 import classNames from '@components/UI/backdrop/backdrop.module.scss';
 
 const Backdrop = ({ show, clicked }) =>
-  show ? <div className={classNames.Backdrop} onClick={clicked} /> : null;
+  show ? (
+    <div className={classNames.Backdrop} tabIndex="-1" onClick={clicked} />
```

Comment:
Any particular reason for going with negetive number here?
In the particular modal , I have not observed any other div with tab index

Reply:
negative tabindex indicating that it will not be reachable via sequential keyboard navigation

Reply:
That's true, does this also mean that we would be unable to access any details in modal through sequential keyboard navigation?
http://web-accessibility.carnegiemuseums.org/code/dialogs/